src/main/java/com/regalii/regaliator/RegaliatorXpayV32.java

```diff
@@ -0,0 +1,97 @@
+package com.regalii.regaliator;
```

Comment:
Same for this, should be on examples directory or a different repo, maybe just include in the README a Gist inside the same GitHub